db/crud.go

```diff
@@ -3211,7 +3211,7 @@ func (db *DatabaseCollectionWithUser) CheckProposedRev(ctx context.Context, doci
 }
 
 // CheckProposedVersion - given DocID and a version in string form, check whether it can be added without conflict.
-func (db *DatabaseCollectionWithUser) CheckProposedVersion(ctx context.Context, docid, proposedVersionStr string, previousRev string) (status ProposedRevStatus, currentVersion string) {
+func (db *DatabaseCollectionWithUser) CheckProposedVersion(ctx context.Context, docid, proposedVersionStr string, previousRev string, proposedHLVString string) (status ProposedRevStatus, currentVersion string) {
```

Comment:
For clarity, would changing this help?

Given that this is only called in one place, is it simpler to have this function take `proposedRev` instead of `proposedVersionStr` and have the beginning look like?

```
proposedVersion, err := ParseVersion(ExtractCVFromProposeChangesRev(rev))
```

Or document in the docstring what these args are, because then `proposedVersionStr` should actually be "proposedCV"?